### DISCUSSION TITLE
Is there any api documentation page for unreal marketplace api ?

### DISCUSSION THREAD

**Discussion by as3mbus** [hugoattal/OrbitalMarket#34] [Q&A] [answered]
hello i am currently working on automation tool to check whether or not we have already bought asset from unreal marketplace.

i already searched through the internet but found nothing,  but this project, as I looked it have unreal scrapper API that able to request unreal marketplace API. 

where / how does you understand / achieve this API request condition ?

**Comment by hugoattal** [comment, marked as answer]
Hey!

Basically, I used the network inspector to check what requests were made while navigating the marketplace.
There's no official documentation to my knowledge...

Here is the kind of link to list products, for example: https://github.com/hugoattal/OrbitalMarket/blob/main/backend/src/scrapper/unreal/api/index.ts#L58

**Comment by as3mbus** [comment]
thank you for your response :) 

what tool do you use for network inspector ?

i'm looking for an api to authenticate a user and check whether this user already logged in or not, if you happen to know about it.

**Reply by hugoattal** [reply]
It's the network inspector available in any browser (Firefox, Chrome...)
Here's a resource I found on it: https://developer.chrome.com/docs/devtools/network/

To check if a user is authenticated or has a marketplace, you might want to check the Plugin Warden module from the Unreal Engine. I don't specifically know how it work, but you might figure it out by looking at its source code.
https://docs.unrealengine.com/5.1/en-US/API/Editor/PluginWarden/

**Reply by as3mbus** [reply]
thanks a lot for your help. I'd tinker around with the information you share
